modules/seqexec/model/src/main/scala/seqexec/model/enum/BatchExecState.scala

```diff
@@ -21,14 +21,29 @@ object BatchExecState {
   case object Stopping extends BatchExecState // Queue was commanded to stop, but at least one sequence is still running.
   case object Completed extends BatchExecState // All sequences in the queue were run to completion.
 
-  implicit val equal: Eq[BatchExecState] = Eq.fromUniversalEquals
-
   implicit val show: Show[BatchExecState] = Show.show {
     case Idle      => "Idle"
     case Running   => "Running"
     case Waiting   => "Waiting"
     case Stopping  => "Stopping"
     case Completed => "Completed"
+  }
 
+  def tag(s: BatchExecState): String = s match {
+    case Idle      => "Idle"
+    case Running   => "Running"
+    case Waiting   => "Waiting"
+    case Stopping  => "Stopping"
+    case Completed => "Completed"
   }
```

Comment:
I assumed this would go into the `BatchExecState` `trait` itself with a 

```scala
```

.  At any rate, `Show` should probably call this if you leave it this way so it isn't duplicated?  We should probably have a 1-hour team meeting about whether `Show` should be defined at all and if so whether it should ever be defined to anything but `Show.fromToString`.  :-)